managed/services/telemetry/config.default.yml

```diff
@@ -49,6 +49,20 @@ telemetry:
       - metric_name: "pmm_server_updates_disabled"
         column: "updates_disabled"
 
+  - id: PMMAgentRandomInfo
+    source: PMMDB_SELECT
+    query: agent_type, status from agents
+    summary: "PMMAgentRandomInfo"
+    json_format:
```

Comment:
nit: it would be good to follow naming schema. e.g. `data_json`

Reply:
changed to data_json